### PR TITLE
Update for ohmynews.com

### PR DESCRIPTION
**Pull request by seia-soto**
Advertisement on [ohmynews.com](ohmynews.com) desktop version including self-hosted ads.

- http://www.ohmynews.com/
- http://www.ohmynews.com/NWS_Web/Series/series_premium_pg.aspx?CNTN_CD=A0002826280&PAGE_CD=ET001&BLCK_NO=1&CMPT_CD=T0016
- http://www.ohmynews.com/NWS_Web/ArticlePage/Life/index.aspx

### CODE REVIEW

File: youslist.txt [hunk 1]
```diff
@@ -6151,14 +6151,17 @@ odysee.com##div.ads__claim-item
 ohli24.net##div#exoall
 ohli24.net##div#exon
 ohli24.net##div.insty
+ohmynews.com###div10Sticky
```
Comment (yous) [inline youslist.txt:6154]: I think this one can be annoying, but it's for sponsoring their own site. I'll let them be.

File: youslist.txt [hunk 2]
```diff
@@ -6151,14 +6151,17 @@ odysee.com##div.ads__claim-item
...
 ohmynews.com##div.ad_footer
 ohmynews.com##div.ad_scroll
-ohmynews.com##div.ad_scroll_rt
 ohmynews.com##div.aside_google_ad
 ohmynews.com##div.wing_gallery_banner
+ohmynews.com##div[class^="ad_scroll_"]
 ohmynews.com##div[class^="aside_ad"]
 ohmynews.com##div[class^="m_ad"]
+ohmynews.com##div[class^="new_ad"]
+ohmynews.com##img[src*="/images/ohmynews/main/main_club_btn.gif"]
```
Comment (yous) [inline youslist.txt:6164]: This one is also for sponsoring their own site.

**Comment by yous** [comment]
Please add this line, came from the second link:
```
ohmynews.com##div.AD-160x600
```

**Comment by seia-soto** [comment]
@yous Changes reflected. May I request you to review again?

**Comment by yous** [comment]
Thanks!

**Pull request merged**